removed @types/luxon as a dependency and moved it to devDependecies (#144)
mrjones2014/yaclt: changelogs/2021-09-10T15-14-33.236-04-00.md

> [IMPROVED] Move @types/luxon to devDependencies instead of having it be a regular dependency {#118}

Reviewer: ```suggestion
[IMPROVED] Move `@types/luxon` to `devDependencies` instead of `dependencies` {#118}
```
